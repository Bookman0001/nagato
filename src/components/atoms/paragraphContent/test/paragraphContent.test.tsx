import { shallow } from 'enzyme'

import ParagraphContent from '../'

test('ParagraphContent', () => {
  const component = shallow(<ParagraphContent>test</ParagraphContent>)
  expect(component).toMatchSnapshot()
})
