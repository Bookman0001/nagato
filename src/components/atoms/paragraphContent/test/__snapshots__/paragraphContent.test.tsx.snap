// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParagraphContent 1`] = `
<styled.div>
  <styled.p>
    test
  </styled.p>
</styled.div>
`;
